docs/input_box.md

```
@@ -0,0 +1,94 @@
+	TITLE="$DIALOG Test"
+	result=$(input_box "Enter your email (dialog):" "user@example.com")
+	echo "You entered: $result"
+
+	DIALOG="read"
+	TITLE="$DIALOG Test"
+	result=$(input_box "Enter a number (read):" "42")
+	echo "You entered: $result"
+fi
+
+# staging/input_box.sh
+#!/bin/bash
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading
null

(MD018, no-missing-space-atx)